source/sass/views/article.scss

```diff
@@ -41,6 +41,7 @@
 .article-summary-share-links {
   position: absolute;
   left: 0;
+  z-index: 99;
```

Comment:
Fixes a bug I found on the article page in which the content div is overlapping the buttons, making some of them unclickable.